SU-HSDO/suhumsci
# Url encode course link

# READY FOR REVIEW

# Summary
- URL Encode course links during import.
- note: the RoboFile.php that is deleted in this pr was unknowingly commited during automated release. I gotta fix that before the next release

# Needed By (Date)
- 7/3

# Urgency
- low

# Steps to Test
1. checkout branch.
1. sync symsys `blt drupal:sync --site=symsys --partial`
1. `drush @symsys.local mim hs_courses`
1. view `node/4321` and click on the course link
1. ensure the courses page you land on has the "Introduction to Optimization (ENGR 62, MS&E 211)" as the first item

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
